addons/website/views/snippets/s_popup.xml

```diff
@@ -72,6 +72,7 @@
                 <we-button data-select-data-attribute="mouseExit">On Exit</we-button>
             </we-select>
             <we-input string="⌙ Delay"
+                      title="Open the pop-up if the user stays on a page for more than X seconds."
```

Comment:
@sbuhl Is not this "too technical" for user to speak about "more than X seconds"?